docs/migration/migrating-enact-apps.md

```diff
@@ -0,0 +1,531 @@
+	return (
+		<Item>
+			{title}
+		</Item>
+	);
+};
+...
+<VirtualGridList
+	...
+	itemRenderer={itemRenderer}
+	...
+/>
+```
+
+#### `VirtualGridList.GridListImageItem`
+This component is replaced by `ui/GridListImageItem`.
+
+#### `VirtualList`
+The `scrollTo` method's `indexToFocus` option has been removed.  Use the `focus` option and
+scroll by `index` or `node`.
+##### Example
+###### 1.x
+```
+...
+const cbScrollTo = () => {
+	this.scrollTo({indexToFocus: 1});
+};
+...
+<VirtualList cbScrollTo={cbScrollTo} ... />
+...
+```
+###### 2.0
+```
+...
+const cbScrollTo = () => {
+	this.scrollTo({focus: true, index: 1});
+};
+...
+<VirtualList cbScrollTo={cbScrollTo} ... />
+...
+```
+The `component` and `data` props are replaced by `itemRenderer`.  `itemRenderer` should return
+the component to render from a set of arbitrary items given an index.
+##### Example
+###### 1.x
+```
+...
+import Item from '@enact/moonstone/Item';
+const items = [];
+const ListItem = ({data, index}) => {
+	const {title} = data[index];
+	return (
+		<Item>
+			{title}
+		</Item>
+	);
+};
+...
+<VirtualList
+	...
+	component={ListItem}
+	data={items}
+	...
+/>
+```
+###### 2.0
+```
+...
+import Item from '@enact/moonstone/Item';
+const items = [];
+const itemRenderer = ({index}) => {
+	const {title} = items[index];
+	return (
+		<Item>
+			{title}
+		</Item>
+	);
+};
+...
+<VirtualList
+	...
+	itemRenderer={itemRenderer}
+	...
+/>
+```
+
+### spotlight
+Selectors targeting the `spotlight` container attributes `data-container-disabled` or `data-container-muted`
+should use `data-spotlight-container-disabled` or `data-spotlight-container-muted` instead.
+
+#### `SpotlightContainerDecorator`
+The `containerId` prop has changed to `spotlightId`.
+##### Example
+###### 1.x
+```
+const MyComponent = ...;
+const MyContainer = SpotlightContainerDecorator(MyComponent);
+<MyContainer containerId="spotlightContainer" />
+```
+###### 2.0
+```
+const MyComponent = ...;
+const MyContainer = SpotlightContainerDecorator(MyComponent);
+<MyContainer spotlightId="spotlightContainer" />
+```
+
+### ui
+
+#### `Group`
+`Group` now requires a unique key for `Object` type data. [Read about keys](https://reactjs.org/docs/lists-and-keys.html#keys)
+for more information.
+
+#### `Holdable`
+This component has been replaced by `ui/Touchable`.
+
+#### `Pressable`
+This component has been replaced by `ui/Touchable`.
+
+#### `Repeater`
+`Repeater` now requires a unique key for `Object` type data. [Read about keys](https://reactjs.org/docs/lists-and-keys.html#keys)
+for more information.
+
+#### `Toggleable`
+The default `prop` for the default HOC configuration is changed from `'active'` to `'selected'`.
+
+#### `Transition`
+`children` is now a required prop for this component.
+
+The `clipHeight` prop has been removed.  The base component's `clipHeight` is now automatically computed base on the
+`Transition`'s initial height.
+
+### webos
+
+#### `LS2Request`
+Import change.
```

Comment:
More detail is required here. I don't remember what the change even was to suggest what detail :)